frontend/app/models/spree/frontend_configuration.rb

```diff
@@ -1,5 +1,7 @@
 module Spree
   class FrontendConfiguration < Preferences::Configuration
     preference :locale, :string, default: Rails.application.config.i18n.default_locale
+
+    preference :require_terms_and_conditions_acceptance, :boolean, default: false
```

Comment:
Could we add a comment here that they'll need to add the actual terms and conditions in `app/views/spree/checkout/_terms_and_conditions.en.html.erb` in their app?